src/components/Attachments/AttachmentCarousel/index.native.js

```diff
@@ -154,17 +132,11 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
 
                             <AttachmentCarouselPager
                                 items={attachments}
-                                isZoomedOut={isZoomedOut}
                                 renderItem={renderItem}
                                 initialIndex={page}
-                                onTap={() => {
-                                    if (!isZoomedOut) {
-                                        return;
-                                    }
-                                    setShouldShowArrows(!shouldShowArrows);
-                                }}
+                                shouldShowArrows={shouldShowArrows}
+                                setShouldShowArrows={setShouldShowArrows}
```

Comment:
This seems like a new pattern that we don't follow. Children should not be able to control parent state. To keep the required change simple add a new prop `onRequestToggleArrows`.

Inside AttachmentCarouselPager

setShouldShowArrows(true / false) becomes onRequestToggleArrows(true / false)
setShouldShowArrows(!shouldShowArrows) becomes onRequestToggleArrows()

IOW, if we pass a param to onRequestToggleArrows we use that param, otherwise we toggle the current value